portal-web/test/functional/com/liferay/portalweb/macros/Questions.macro

```diff
@@ -562,6 +592,34 @@ definition {
 		}
 	}
 
+	macro viewUnlockedQuestion {
+		AssertElementNotPresent(
+			key_questionTitle = "${questionTitle}",
+			locator1 = "Questions#QUESTIONS_LOCKED_ICON");
+
+		for (var caretName : list "caret-top,caret-bottom") {
+			AssertVisible(
+				key_caretName = "${caretName}",
+				locator1 = "Questions#VOTE_BUTTON");
+		}
+
+		AssertVisible(locator1 = "Questions#QUESTIONS_BELL_BUTTON");
+
+		AssertVisible(locator1 = "Icon#MOVE_TO_RECYCLE_BIN");
+
+		AssertVisible(locator1 = "Button#EDIT");
+
+		AssertVisible(locator1 = "Button#REPLY");
```

Comment:
@Jeremy-Ch when you have redundant assertions you could turn this into a list. That way you can write cleaner code